### ISSUE TITLE
current pattern does not work

### ISSUE THREAD

**Issue by veffy58** [Meckazin/ChromeKatz#21]
current pattern does not work i just put the new pattern that i found still says failed to find pattern

**Comment by yousufs85** [comment]
Yes It is not working any more

**Comment by tangentbordskrigaren** [comment]
Correct. The pattern for credentials does not work anymore. However, the cookie part still works :)

**Comment by Meckazin** [comment]
CredentialKatz is deprecated, see README for more information
